src/main/java/com/endava/soa_5ed/task4/collections/Order.java

```diff
@@ -0,0 +1,54 @@
+package com.endava.soa_5ed.task4.collections;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ThreadLocalRandom;
+
+public class Order {
+    public List<Item> items;
+    private final long id;
+    private final LocalDate date;
+    private User user;
+
+    public Order() {
```

Comment:
make is private please so no one could use it outside of this class

Reply:
Done